mod/uexport.php

```diff
@@ -109,7 +109,7 @@ function uexport_account($a) {
 	);
 
 	$group_member = _uexport_multirow(
-		sprintf("SELECT * FROM `group_member` WHERE uid = %d", intval(local_user()))
+		sprintf("SELECT `group_member`.`gid`, `group_member`.`contact-id` FROM `group_member` INNER JOIN `group` ON `group`.`id` = `group_member`.`gid` WHERE `group`.`uid` = %d", intval(local_user()))
```

Comment:
Wouldn't ```SELECT *```` be better in this special case of exporting user data? We only have to be sure that the receiver can handle importing fields that aren't existing any more.

Reply:
Until we remove the `id` column, there is a risk of collision with the receiver `group_member` table.

Reply:
Ah, you are right. I hadn't seen that there is now a joined query there.